### PR TITLE
Monitor metrics from multiple projects

### PR DESCRIPTION
**Pull request by ashiddo11**
Fixes issue #55

### CODE REVIEW

File: stackdriver_exporter.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 
 var (
 	projectID = kingpin.Flag(
-		"google.project-id", "Google Project ID ($STACKDRIVER_EXPORTER_GOOGLE_PROJECT_ID).",
+		"google.project-id", "Comma seperated Google Project IDs ($STACKDRIVER_EXPORTER_GOOGLE_PROJECT_ID).",
```
Comment (SuperQ) [inline stackdriver_exporter.go:23]: Minor nit:

```suggestion
		"google.project-id", "Comma separated list of Google Project IDs ($STACKDRIVER_EXPORTER_GOOGLE_PROJECT_ID).",
```

**Comment by SuperQ** [comment]
This needs to be re-based on master.

**Comment by SuperQ** [comment]
Closing in favor of https://github.com/prometheus-community/stackdriver_exporter/pull/105

**Comment by inkaabi** [comment]
Hello, it is ok to take multiple project but it still the problem to pass multiple credential? any idea please

**Pull request closed without merging**